### PR TITLE
Add Changelog and update Installing, FAQ

### PR DESCRIPTION
**Pull request by dhaavi**
- Add and improve data fetching and processing
- Improve FAQ rendering
- Add changelog page
- Update linux installers and dependencies
- Update notices

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -13,6 +13,6 @@ Get started by:
 
 ### Frequently Asked Questions
 
-You can [search through all entries on GitHub](https://github.com/issues?q=archived%3Afalse+user%3Asafing+sort%3Aupdated-desc+label%3Afaq).
+You can your browser search on [this page]({{ site.faq_url }}) or [search through all entries on GitHub](https://github.com/issues?q=archived%3Afalse+user%3Asafing+sort%3Aupdated-desc+label%3Afaq).
```
Comment (davegson) [inline index.md:16]: wording is not right here ;) but push to `master` for this small fix after I merged

File: spn/index.md [hunk 2]
```diff
@@ -3,4 +3,6 @@ title: SPN
 layout: base
 ---
 
-SPN Base Page
+The expanded SPN documentation is in progress.
+
+You can find more information about the SPN [on our website]({{ site.spn_url }})
```
Comment (davegson) [inline spn/index.md:8]: maybe something like "will be added in the future"

"in progress" quickly becomes a lie when priorities shift

**Review by davegson — APPROVED** [review]
good from  my side (excluding the Update System docs, since @Raphty will do that iirc)

**Comment by dhaavi** [comment]
Also approved by @Raphty.

**Pull request merged**